sim/rogue/items_sets_pve.go

```diff
@@ -156,8 +157,9 @@ var ItemSetNightSlayerThrill = core.NewItemSet(core.ItemSet{
 				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
 					core.Each(affectedSpells, func(spell *core.Spell) { spell.Cost.Multiplier += 100 })
 				},
+				//Poisoned Knife is ignored
 				OnCastComplete: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell) {
-					if aura.RemainingDuration(sim) == aura.Duration || spell.DefaultCast.Cost == 0 {
+					if aura.RemainingDuration(sim) == aura.Duration || spell.DefaultCast.Cost == 0 || spell.ActionID.SpellID == 425012 {
```

Comment:
Would you mind updating this to use a Poisoned Knife SpellCode?

Reply:
done